governance/api.go

```diff
@@ -96,6 +96,28 @@ func (api *GovernanceKlayAPI) GasPriceAt(num *rpc.BlockNumber) (*hexutil.Big, er
 	}
 }
 
+// or returns gas price of txpool if the block is pending block.
+func (api *GovernanceKlayAPI) GetRewards(num *rpc.BlockNumber) (*reward.RewardSpec, error) {
+	if num == nil || *num == rpc.LatestBlockNumber {
+		header := api.chain.CurrentHeader()
+		config := api.chain.Config()
+		return reward.GetBlockReward(header, config)
+	}
+
+	header := api.chain.GetHeaderByNumber(num.Uint64())
+	pset, err := api.governance.ParamsAt(num.Uint64())
+	if err != nil {
+		return nil, err
+	}
+	config := pset.ToChainConfig()
+	config.IstanbulCompatibleBlock = api.chain.Config().IstanbulCompatibleBlock
+	config.LondonCompatibleBlock = api.chain.Config().LondonCompatibleBlock
+	config.EthTxTypeCompatibleBlock = api.chain.Config().EthTxTypeCompatibleBlock
+	config.MagmaCompatibleBlock = api.chain.Config().MagmaCompatibleBlock
+	config.KoreCompatibleBlock = api.chain.Config().KoreCompatibleBlock
```

Comment:
Let's figure out how to simplify this idiom later.

Reply:
Let's address this in future PR